src/workflow_tasks.rs

```diff
@@ -176,10 +184,10 @@ impl WorkflowTaskManager {
     /// Evict a workflow from the cache by its run id and enqueue a pending activation to evict the
     /// workflow. Any existing pending activations will be destroyed, and any outstanding
     /// activations invalidated.
-    ///
     /// Returns that workflow's task info if it was present.
     pub fn evict_run(&self, run_id: &str) -> Option<WorkflowTaskInfo> {
         debug!(run_id=%run_id, "Evicting run");
+        self.cache_manager.write().touch(run_id);
```

Comment:
Why do we need to touch the LRU here? Can you please document the reason?

Reply:
It got messed up in the process of addressing comments, it should be remove here.